src/resources/generated/tax_rate.rs

```diff
@@ -159,14 +159,14 @@ pub struct CreateTaxRate<'a> {
 }
 
 impl<'a> CreateTaxRate<'a> {
-    pub fn new(display_name: &'a str, percentage: f64) -> Self {
+    pub fn new(display_name: &'a str, inclusive: bool, percentage: f64) -> Self {
```

Comment:
This is a breaking change in the generated code I need to fix before removing this pr from draft